apsuite/commisslib/emit_exchange/meas_bo_tune_bpm.py

```diff
@@ -33,26 +32,26 @@ def __init__(self):
 
     def __str__(self):
         """."""
-        ftmp = '{0:24s} = {1:9.3f}  {2:s}\n'.format
-        dtmp = '{0:24s} = {1:9d}  {2:s}\n'.format
-        stmp = '{0:24s} = {1:9s}  {2:s}\n'.format
-
-        stg = ''
-        stg += stmp('event', self.event, '')
-        stg += stmp('event_mode', self.event_mode, '')
-        stg += ftmp('event_delay', self.event_delay, '[us]')
-        stg += stmp('trigger_source', self.trigger_source, '')
+        ftmp = "{0:24s} = {1:9.3f}  {2:s}\n".format
```

Comment:
@fernandohds564 , why are you replacing single quote with double quote characters? you will be choosing a differente standard we adopt in other repos. what is the rationale ?

Reply:
@xresende, please check my PR in dev-packages on linter and formatter configurations.